.github/workflows/merge-major-version.yml

```diff
@@ -16,10 +16,30 @@ jobs:
                   node-version: '14'
 
             - name: Checkout repository
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
+              with:
+                fetch-depth: 0
 
-            - name: Open PR
+            - name: Prepare branch to merge
               run: |
-                  cd scripts/create_pull_request
-                  npm install
-                  node index.js ${{ secrets.ZOWE_ROBOT_TOKEN }}
+                  git config --global user.email "zowe-robot@users.noreply.github.com"
+                  git config --global user.name "Zowe Robot"
+                  git checkout master
```

Comment:
Is it necessary to checkout master here?

Reply:
Well probably not, but it fails then if you try to run it from a different branch.

Reply:
Ok, got it

Reply:
That's the reason why I would rather keep it.

Reply:
Yes, I'm ok with keeping it then. The PR looks good to me